app/src/androidTest/java/ch/epfl/sdp/drone3d/ui/mission/MapActivityTest.kt

```diff
@@ -72,4 +90,37 @@ class MapActivityTest {
         imageButton.perform(click())
         intended(hasComponent(MainActivity::class.java.name))
     }
+
+    @Test
+    fun denyLocatePositionWorks() {
+        `when`(context.checkSelfPermission(ACCESS_FINE_LOCATION))
+            .thenReturn(
+                PackageManager.PERMISSION_DENIED
+            )
+
+        activityRule.scenario.onActivity {
+            it.onRequestPermissionsResult(
+                0,
+                arrayOf("android.permission.ACCESS_FINE_LOCATION"),
+                intArrayOf(-1)
+            )
+        }
+
+    }
+
+    @Test
+    fun allowLocatePositionWorks() {
```

Comment:
I'm not sure what those tests are testing. Can you explain it to me please ?

Reply:
It does not test anything yet. For the moment it is just here to increase coverage but I'm trying to figure out a way to make it test something.